jenkins_jobs/modules/triggers.py

```diff
@@ -1172,6 +1172,8 @@ def github_pull_request(registry, xml_parent, data):
         in the pull request will trigger a build (default false)
     :arg str skip-build-phrase: when filled, adding this phrase to
         the pull request title or body will not trigger a build (optional)
+    :arg str black-list-commit-author: When filled, pull request commits from this user(s)
```

Comment:
The type should probably be "list" and the name of the key should rather become "black-list-commit-authors".

Sorry for being annoying, I'm just thinking about upstreaming.

Reply:
No of course not, thanks for noticing :)